mne/viz/utils.py

```diff
@@ -509,6 +509,8 @@ def _helper_raw_resize(event, params):
 
 def _plot_raw_onscroll(event, params, len_channels=None):
     """Interpret scroll events"""
+    if 'fig_selection' in params:  # Not supported in selection mode.
+        return
```

Comment:
Shouldn't scroll up go to previous selection, scroll down go to the next?

Actually same with up/down arrow keys